### DISCUSSION TITLE
Upgrade to NC25?

### DISCUSSION THREAD

**Discussion by ShawkMusic** [nextcloud/all-in-one#1360] [Questions] [answered]
I need to upgrade to NC 25. Nextcloud 25.0.1 has been released and I'm in the beta channel, why isn't it updating? Am I misunderstanding the update policy? I understand that new updates will be held back to ensure stability, but isn't that only for the stable channel? Shouldn't I be getting updates earlier if im in the beta channel? If not, when am I getting it and what's the point of a beta channel? What's up with that?

**Comment by szaimen** [comment]
Hi, see https://github.com/nextcloud/all-in-one/issues/1280

**Comment by ShawkMusic** [comment]
That doesn't answer my question.

**Comment by szaimen** [comment, marked as answer]
We cannot upgrade until the apps are not released and the bugs not fixed. The beta channel is not for testing Nextcloud versions. It will get released to AIO when it is ready.

**Comment by szaimen** [comment]
Hey, so I thought a bit about your request and indeed we should provide some managable way for people to upgrade to new Nextcloud version before we think it is ready. For that I've created https://github.com/nextcloud/all-in-one/pull/1363. You will then be able to run the built-in updater in order to upgrade early to new releases. See https://docs.nextcloud.com/server/latest/admin_manual/maintenance/update.html#using-the-command-line-based-updater

**Reply by szaimen** [reply]
The command in the end will then something like `sudo docker exec --user www-data -it nextcloud-aio-nextcloud php updater/updater.phar`

**Reply by jlmanohar** [reply]
Is nextcloud 25 in beta ? I see 25.0.1 in the main changelog on nextcloud's website.
